src/main/resources/view/MemeGridCard.fxml

```diff
@@ -33,7 +33,7 @@
             <children>
            <HBox alignment="CENTER_RIGHT" prefHeight="24.0" prefWidth="240.0">
              <ImageView fx:id="likeIcon" cache="true" cacheHint="SPEED" fitHeight="11.0" fitWidth="13.0" pickOnBounds="true" preserveRatio="true">
-               <Image requestedHeight="11.0" requestedWidth="13.0" smooth="true" url="@../images/like_icon.png" />
+               <Image requestedHeight="11.0" requestedWidth="13.0" smooth="true" url="@../../../../docs/images/like_icon.png" />
```

Comment:
can try generating the jar and see if this actl works? im not sure if it has the docs dir in the jar file

Reply:
am tired of this IntelliJ shenanigan... Fixed.